src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGridInternal/PropertyDescriptorGridEntryAccessibleObjectTests.cs

```diff
@@ -96,12 +111,30 @@ public GridEntryAccessibleObject TestPropertyDescriptorGridEntryAccessibleObject
             }
         }
 
-        internal class TestEntity
+        private class TestEntity
         {
             public Font FontProperty
             {
                 get; set;
             }
         }
+
+        private class TestDropDownHolder : PropertyGridView.DropDownHolder
+        {
+            private Control _component;
+
+            public TestDropDownHolder(PropertyGridView psheet, Control component)
```

Comment:
The variable names look out of sync with declaring types
```suggestion
            public TestDropDownHolder(PropertyGridView propertyGridView , Control control)
```